tests/python/relay/test_ir_parser.py

```diff
@@ -76,6 +76,7 @@ def graph_equal(lhs, rhs):
 
 def roundtrip(expr):
     x = relay.fromtext(expr.astext())
+    import pdb; pdb.set_trace()
```

Comment:
remove

Reply:
more debugging for last failing test case will remove when I'm done